CMakeLists.txt

```diff
@@ -397,6 +398,9 @@ endif ()
 if (DISABLE_TTF)
     target_compile_options(libopenrct2 PUBLIC -DNO_TTF)
 endif ()
+if (DISABLE_VERSION_CHECKER)
+    target_compile_options(libopenrct2 PUBLIC -DDISABLE_VERSION_CHECKER)
```

Comment:
The modern way would be to use a generator expression then you wouldn't need the full if block. 
`target_compile_definitions(libopenrct2 PUBLIC $<${DISABLE_VERSION_CHECKER}:DISABLE_VERSION_CHECKER>)`. Also in general you should use compile definitions for compile definitions as compile options are for compiler options.

Reply:
In this case, they were adhering to what was there already, though. Sounds like it would be good to improve these things for the entire project in several sweeps.

Reply:
> The modern way would be to use a generator expression then you wouldn't need the full if block. `target_compile_definitions(libopenrct2 PUBLIC $<${DISABLE_VERSION_CHECKER}:DISABLE_VERSION_CHECKER>)`. Also in general you should use compile definitions for compile definitions as compile options are for compiler options.

I didn't want to be nitpicky as that is how our current CMake setup works but yeah its sort of wrong.